gateway/ln-gateway/src/client.rs

```diff
@@ -106,8 +106,8 @@ impl GatewayClientBuilder {
         .map_err(GatewayError::ClientStateMachineError)
     }
 
+    // TODO(tvolk131): Move this to the DB code.
```

Comment:
What does this todo mean? Are you referring to the `DbManager`?

Reply:
Yes. Sorry, that comment was a bit unclear, I updated it.